frontend/src/views/Lobby.tsx

```diff
@@ -48,6 +53,15 @@ function LobbyPage() {
     console.log(user);
   };
 
+  const handleClick = (): Promise<void> => {
+    const request = { roomId: currentRoomId, User: currentUser };
+    return axios.post<void>('/api/v1/start', request)
+      .then(() => { console.log('redirect to game'); })
+      .catch((err) => {
+        throw axiosErrorHandler(err);
```

Comment:
Use the `setError` function (or add it if it does not exist), so the user can see that / when an error occurred. (See my image below for what happens otherwise -- it just prints to the console, but the user does not get any notice that it failed.)

<img width="1280" alt="Error on Starting Game" src="https://user-images.githubusercontent.com/7987279/95029002-b7c61480-0659-11eb-928f-ced44a0eec5d.png">